src/store/printer/getters.ts

```diff
@@ -1006,9 +1006,7 @@ export const getters: GetterTree<PrinterState, RootState> = {
             'print_duration' in state.print_stats &&
             'current_file' in state &&
             'estimated_time' in state.current_file &&
-            state.print_stats.print_duration > 0 &&
-            state.current_file.estimated_time > 0 &&
```

Comment:
i think the estimated_time should be checked. because if this is missing in the gcode metadata, mainsail would display a negative print_duration here.

Reply:
Yeah, good point - I added the check for estimated_time back in.